lib/presentation/widgets/already_have_an_account.widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/custom_theme.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:water_tracker/generated/locale_keys.g.dart';
+import 'package:water_tracker/presentation/screens/login_screen/login_screen.dart';
+
+class AlreadyHaveAnAccountWidget extends StatelessWidget {
+  const AlreadyHaveAnAccountWidget({Key? key}) : super(key: key);
+
+  static const spaceTextSpan = TextSpan(text: ' ', style: TextStyle(fontSize: 16, color: Colors.black));
+  static const notHaveAccountPadding = EdgeInsets.all(8.0);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: notHaveAccountPadding,
```

Comment:
lets decide, will we move only int as padding static const variable, or EdgeInsets instance?
f.e., on 1 screen u have
`static const notHaveAccountPadding = EdgeInsets.all(8.0);`
on other:
`static const notHaveAccountPadding = 8.0;`
not make sense to me, so need to define 1 way of doing it. i prefer 1st, with EdgeInsets

Reply:
like this example - tatic const notHaveAccountPadding = EdgeInsets.all(8.0);